src/createSliceSaga.ts

```diff
@@ -136,23 +141,32 @@ export function createSliceSaga<
 
   caseSagasNames.forEach((sagaName) => {
     const type = getType(name, sagaName);
-    let prepareCallback: PrepareAction<any> | undefined;
-    actionCreators[sagaName] = prepareCallback
-      ? createAction(type, prepareCallback)
-      : createAction(type);
-
-    sagas.push(
-      call(
-        sagaType === SagaType.Normal
-          ? caseSagas[sagaName]
-          : createSaga(sagaType, type, caseSagas[sagaName]),
-      ),
-    );
+
+    actionCreators[sagaName] = createAction(type)
```

Comment:
I dont know why this variable was here because it never gets defined. I removed it.

Maybe Im missing something, you tell me :)

Reply:
`prepareCallback` is a variable in the `@reduxjs/toolkit`, which gives me the main idea of ​​this library. When I worked I forgot it. Thank you for fixing it 😂